tests/RobotFramework/tests/cumulocity/firmware/firmware_operation.robot

```diff
@@ -1,34 +1,28 @@
 *** Settings ***
 Resource            ../../../resources/common.resource
-Library             Cumulocity
 Library             ThinEdgeIO
+Library             Cumulocity
```

Comment:
Revert the Libary import sequence (not change is necessary)

```
Library             Cumulocity
Library             ThinEdgeIO
```

Reply:
Resolved by d2c2cff